ddtrace/contrib/kombu/patch.py

```diff
@@ -154,5 +158,7 @@ def traced_publish(func, instance, args, kwargs):
         # set analytics sample rate
         s.set_tag(ANALYTICS_SAMPLE_RATE_KEY, config.kombu.get_analytics_sample_rate())
         # run the command
-        propagator.inject(s.context, args[HEADER_POS])
+        if config.kombu.distributed_tracing_enabled:
```

Comment:
I didn't see a check here for config, unlike the extraction side, so I put it here